website/docs/tutorials/react-native.md

```diff
@@ -60,120 +76,100 @@ const Inbox = ({ messages, markAsRead, username }) => {
+There are several ways to render translations: You may use the [`Trans`](/docs/ref/react.md#trans) component or the [`useLingui`](/docs/ref/react.md#withi18n) hook together with the `t` macro. When you change the active locale or load new messages, all the components that consume the Context provided by `I18nProvider` will re-render, making sure the UI shows the correct translations.
 :::
 
-## Internationalization Outside of React Components
+## Internationalization outside of React
 
-Until now, we have covered the [`Trans`](/docs/ref/react.md#trans) macro and the `I18n` render prop component. Using them will make sure our components are always in sync with the currently active locale.
+Until now, we have covered the [`Trans`](/docs/ref/react.md#trans) macro and the `useLingui` hook. Using them will make sure our components are always in sync with the currently active locale and message catalog.
 
-However, often you'll need to show localized strings outside of React, for example when you want to show a toast from some business logic code. In that case you'll also need access to the `i18n` object, but you don't want to pass it around from some component's props. At this point, we need to turn our attention to the `@lingui/core` package, namely the [`setupI18n`](/docs/ref/core.md#setupi18n) method which returns an `i18n` object.
+However, often you'll need to show localized strings outside of React, for example when you want to show an Alert from some business logic code.
 
-```jsx
-import { setupI18n } from '@lingui/core';
-
-// this file is generated by the cli
-import enMessages from './locale/en/messages.js';
+In that case you'll also need access to the `i18n` object, but you don't want to pass it around from some React component.
+Lingui uses a default i18n object instance that you can import as follows:
 
-// import this constant as get translations from it outside of React
-export const i18n = setupI18n({
-  locale: 'en',
-  catalogs: {
-    en: enMessages,
-  },
-});
+```
+import { i18n } from '@lingui/core';
 ```
 
-As explained before, [`I18nProvider`](/docs/ref/react.md#i18nprovider) creates an instance of the `i18n` object automatically and passes it to [`Trans`](/docs/ref/react.md#trans) components through React Context. Since we created the `i18n` instance by ourselves, we need to pass it to the [`I18nProvider`](/docs/ref/react.md#i18nprovider) as a prop. This way we tell it not to create a new instance but use the one we provide, like this:
+This instance is the source of truth for the active locale and the `t` macro implicitly uses this instance, unless you pass custom instance like this `` t(i18n)`some message` ``.
 
-```jsx
-<I18nProvider i18n={i18n} locale="en">
-  <YourRootComponent someProp="someValue" />
-</I18nProvider>
-```
+What that means is that in non-react code you can just call `` t`this will be translated` `` which will render the translation for currently active locale.
```

Comment:
Here could be a caveat. Many people start using this in non-reactive (non-callable) contexts such as constants

```ts
// colors.constatns.ts
const colors = {
  blue: t`Blue`,
  red: t`Red`,
}
```

So obviously this will not react for the locale changes and in general it's misusage. 

The correct way is to define message for later use with `defineMessage` or `msg` and than use it later in the components/alerts/whatever

```ts
// colors.constatns.ts
const colors = {
  blue: msg`Blue`,
  red: msg`Red`,
} as const


function showAlert(colorKey: keyof typeof colors) {
  alert(i18n._(colors[colorKey]));
}
```

There is a paragraph about that in the `website/docs/tutorials/react-patterns.md`

Reply:
haha, right, I didn't realize someone could interpret it that way 😅 
I will update that